client/src/app/util/AuthContext.js

```diff
@@ -13,6 +13,30 @@ export function AuthProvider(props) {
   const [loggedIn, setLoggedIn] = useState(false);
   const [userProfile, setUserProfile] = useState(null);
 
+  React.useEffect(()=>{
```

Comment:
small clean code note: import `useEffect` in line 1, and then replace this with just `useEffect()`